Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1457,10 +1497,18 @@ def create_ticket_command(client: Client, args: dict) -> tuple[str, dict, dict,
     else:
         additional_fields_keys = list(args.keys())
 
+    instance_url = demisto.params().get("url", "").rstrip('/')
```

Comment:
dont yo need to pass here an argument that say if yo been called by the quickaction/need to create a mirroring object and do it only then?
see:
https://github.com/demisto/content/pull/39826/files#r2090682497